homestar-runtime/src/network/swarm.rs

```diff
@@ -57,6 +57,18 @@ pub(crate) async fn new(settings: &settings::Node) -> Result<Swarm<ComposedBehav
     // Listen-on given address
     swarm.listen_on(settings.network.listen_address.to_string().parse()?)?;
 
+    // Dial bootstrap nodes specified in settings. Failure here shouldn't halt node startup.
+    for bootstrap_addr in &settings.network.bootstrap_addresses {
+        swarm
+            .dial(bootstrap_addr.clone())
```

Comment:
so we dial now, but shouldn't we add bootstrap nodes, that succeed in dialing, as explicit peers?

Reply:
I think this is reasonable for us to be doing at this point, but in this case I think we should probably rename it to "trusted_peers" or similar. Bootstrap nodes for larger (untrusted) networks normally are still included in peer scoring so that they can eventually leave a submesh after a new node builds it's own mesh up. I think it is best to avoid any potential confusion between bootstrap peers and (what this change will do) trusted peers.

Reply:
yep, agreed on the naming convention(s).